src/main/java/net/imglib2/outofbounds/OutOfBoundsConstantValue.java

```diff
@@ -85,4 +97,12 @@ final public T get()
 	{
 		return copy();
 	}
+
+	private static < T > Supplier< T > makeSupplierFrom( final T t ) {
+		if ( t instanceof Type< ? > ) {
+			final Type< ? > type = ( Type< ? > ) t;
+			return () -> ( T ) type.copy();
+		}
+		return () -> t;
+	}
```

Comment:
Please use imglib2 formatter

Reply:
Sorry, my bad. Fixed in 13b05322174be24818497924f42b189f163dff14